usr/share/rear/conf/default.conf

```diff
@@ -2304,15 +2304,16 @@ USE_CFG2HTML=
 # If SKIP_CFG2HTML is enabled, skip this script (backward compatibility) - the var will become obsolete in rear-1.18
 # SKIP_CFG2HTML=
 
-# Simplify bonding setups by configuring always the first active device of a bond
+# Simplify bonding setups by configuring always the first active device of a
+# bond, except when mode is 4 (IEEE 802.3ad policy)
 SIMPLIFY_BONDING=
 
 # Simplify bridge setups by configuring always the first active device of a bridge
 SIMPLIFY_BRIDGE=
 
-# Simplify team setups by configuring always the first active device of a team
-# Current implementation only supports this mode
-#SIMPLIFY_TEAM=
+# Simplify team setups by configuring always the first active device of a team,
+# except when runner is 'lacp' (IEEE 802.3ad policy)
+SIMPLIFY_TEAM=
```

Comment:
@rmetrich 
this cannot work because in
usr/share/rear/rescue/GNU/Linux/310_network_devices.sh
you use `SIMPLIFY_TEAMING`